buildpipeline/DotNet-CoreFx-Trusted-OSX.json

```diff
@@ -1,5 +1,29 @@
 {
   "build": [
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": false,
+      "displayName": "Run AgentTools/Begin.ps1",
```

Comment:
OSX should have begin/end.sh files

Reply:
Fixed. Updated OSX and Linux definitions.